client/hosting/sites/components/dotcom-style.scss

```diff
@@ -147,15 +147,6 @@
 	table.dataviews-view-table thead .dataviews-view-table__row th {
 		border-bottom-color: var(--color-border-secondary);
 
```

Comment:
I suppose this style file is only used in that dashboard but how can we know that?

Reply:
Do you mean, how do we know if the scope of this rule is limited to the dashboard? I see a parent class above:

`.wpcom-site .main.a4a-layout.sites-dashboard.sites-dashboard__layout .sites-overview {`

Reply:
Ok, that's helpful thank you. 

I was wondering if there's a convention from the files names or something to know the scope of each file (even if it's just a convention)